python/setup.py

```diff
@@ -421,31 +421,6 @@ def get_outputs(self):
         default_version.replace('-SNAPSHOT', 'a0')
 
 
-# See https://github.com/pypa/setuptools_scm#configuration-parameters
-scm_version_write_to_prefix = os.environ.get(
-    'SETUPTOOLS_SCM_VERSION_WRITE_TO_PREFIX', setup_dir)
-
-
-def parse_git(root, **kwargs):
-    """
-    Parse function for setuptools_scm that ignores tags for non-C++
-    subprojects, e.g. apache-arrow-js-XXX tags.
-    """
-    from setuptools_scm.git import parse
-    kwargs['describe_command'] =\
-        'git describe --dirty --tags --long --match "apache-arrow-[0-9]*.*"'
-    return parse(root, **kwargs)
-
-
-def guess_next_dev_version(version):
-    if version.exact:
-        return version.format_with('{tag}')
-    else:
-        def guess_next_version(tag_version):
-            return default_version.replace('-SNAPSHOT', '')
-        return version.format_next_version(guess_next_version)
-
-
 with open('README.md') as f:
     long_description = f.read()
```

Comment:
This can be removed as well?

Reply:
Some other variables that can be cleaned up are `install_requires`, `setup_requires` below, and `default_version` a bit above